graphql/achievement_template/mutations.ts

```diff
@@ -39,8 +38,8 @@ class AchievementTemplateCreateInput implements AchievementTemplateCreate {
 
     @Field()
     condition: string;
-    @Field((type) => JSONResolver)
-    conditionDataAggregations: ConditionDataAggregations;
+    @Field()
+    conditionDataAggregations: string;
```

Comment:
I wasn't able to generate the graphql json scalar via node. Therefore, I switched the input type to string and move the parsing to the resolver function.